roadmap/cs-roadmap-2023.md

```diff
@@ -0,0 +1,93 @@
+timeframe.
+
+We plan to focus and achieve these goals:
+
+- Increase active user satisfaction and adoption with improvement’s to UX
+- Decrease customer issues and increase adoption of VNET scenarios
+- Reduce resource consumption and improve security without persistent storage with Ephemeral
+  sessions
+- Continued fraud reduction and enforcement
+- Continue ACI migrations to further improve customer experience and availability
+
+### Tenets
+
+These are the Azure Cloud Shell guiding principles we use to evaluate and prioritize Cloud Shell
+work activities:
+
+- Cloud-based: Azure Cloud Shell is a cloud-based environment that provides users with access to a
+  range of tools and resources, including bash, PowerShell, Azure CLI, and Azure PowerShell.
+- Secure: Azure Cloud Shell is designed with security in mind, and provides a secure, isolated
+  environment for users to work in.
+- Accessible: Azure Cloud Shell is accessible from any device with an internet connection, making it
+  easy for users to manage their resources from anywhere.
+- Reliable: Azure Cloud Shell is reliable, with built-in redundancy and automatic failover.
+- Efficient: Azure Cloud Shell is designed to be efficient, with always-up-to-date tools.
+- Cost-effective: Azure Cloud Shell is cost-effective, with no additional charges for the service
+  beyond the cost of the underlying Azure resources used by the user. It also helps to reduce costs
+  by allowing users to automate tasks and work more efficiently.
+
+Based on our tenets, these are the investment priorities for 2023.
+
+### Improved User Interface
+
+The user interface refresh will help improve the customer experience, accessibility, and
+functionality of Cloud Shell. This will make it easier for users to navigate and find the tools and
+resources they need. Benefits include:
+
+- Improved user interface to be more modern and intuitive.
+- Accessibility improvements for users who may have disabilities or impairments by improving color
+  contrast, adjusting font sizes and other accessibility improvements.
+- Support for enhanced functionality
+
+### Ephemeral Sessions
+
+Ephemeral sessions allow users to manage azure resources faster without the overhead of creating and
+associating Azure storage accounts. These temporary and disposable sessions are best for scenarios
+where customers need to perform ad-hoc tasks without the need to persist and data or files between
+sessions. In addition, this provides a layer of security to scenarios to ensure that data or files
+created during the session are not accessible to other users or processes after the session ends.
+
+Benefits include:
+
+- Allow customers to leverage Cloud Shell without a linked storage account
+- Any saved files will be deleted with the container session
+- Can be leveraged by partners to ensure that a customer can obtain a Cloud Shell session even if a
+  storage account has never been configured
+- Benefits command injection scenarios
```

Comment:
Makes it easier to inject commands? What do you really mean?
```suggestion
- Benefits command injection scenarios
```

Reply:
Great catch - this really doesn't make sense - and I don't think improving helps communicate the message for ephemeral -- so im removing